package.json

```diff
@@ -20,7 +20,8 @@
     "lint": "eslint src",
     "check": "flow check",
     "build": "gulp dist",
-    "start": "node ./scripts/start.js",
+    "start": "gulp dist && node ./scripts/start.js",
```

Comment:
Yea the only reason I changed this, was because i assumed that running start would actually update the code and then run it, and i didnt want anyone else to make this mistake.  But im cool with changing it back